user_guide.md

```diff
@@ -290,6 +296,18 @@ Submit it
 ks apply ${KF_ENV} -c ${CNN_JOB_NAME}
 ```
 
+Delete it
+
+```
+ks delete ${KF_ENV} -c ${CNN_JOB_NAME}
+```
+
+Monitor it (Noted that tf-cnn job is also a tfjobs.)
```

Comment:
Move it above delete too. And link to the doc  https://github.com/kubeflow/tf-operator#monitoring-your-job